### PR TITLE
WID 848 Modal Improvement

### PR DESCRIPTION
**Pull request by gopida**

### CODE REVIEW

File: modals/modals-3.0.4/modal-standard.js [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+
+class StandardModal {
+    constructor(language, theme){
+        this.language = language;
+        this.theme = theme;
+    }
+
+    getContent() {
+        const translation = new Translation().get(this.language);
+        return `<div class="sezzle-checkout-modal-hidden">
```
Comment (aholdahl) [inline modals/modals-3.0.4/modal-standard.js:11]: In LT version, you've removed this tag layer. I see in modal.js that you are assigning class name to sezzle-modal-container, so I believe this is the one that needs updated?

Comment (gopida) [reply modals/modals-3.0.4/modal-standard.js]: yes thanks, updated

File: modals/index.html [hunk 2]
```diff
@@ -1 +1,7 @@
-<div id="sezzle-modal-container"></div>
\ No newline at end of file
+<div id="sezzle-modal-container">
+    <div class="sezzle-modal">
+        <button tabIndex="0" aria-label="Close Sezzle Modal" class="close-sezzle-modal" role="button"></button>
```
[the remark this answers is not in the document]
Comment (gopida) [reply modals/index.html]: This file is only for local testing. And we have only restructured the divs here, so this won't couple modal and js.

File: gulpfile.js [hunk 3]
```diff
@@ -547,6 +547,7 @@ gulp.task('uploadtracker', function () {
     }))
 });
 
+gulp.task('buildmodal', gulp.series('cleanmodal', 'csscompile-modal', 'minify-js', 'minify-modal'));
```
[the remark this answers is not in the document]
Comment (gopida) [reply gulpfile.js:550]: Yes, added this gulp command to build modal locally. `npx gulp buildmodal` will build the modal in `dist` dir.

File: modals/index.html [hunk 4]
```diff
@@ -1 +1,9 @@
-<div id="sezzle-modal-container"></div>
\ No newline at end of file
+<div id="sezzle-modal-container">
+    <div class="sezzle-modal">
```
Comment (aholdahl) [inline modals/index.html:2]: Sorry, just noticed how different this is from the original - missing containers/class names, and button/logo are switched. Based on the original layout, it should be:
```
<div id="sezzle-modal-container">
	<div class="sezzle-checkout-modal-hidden">
		<div class="sezzle-modal">
			<div class="sezzle-modal-content" tabIndex="0">
				<div class="sezzle-logo" title="Sezzle"> </div>
				<button aria-label="Close Sezzle Modal" class="close-sezzle-modal" role="button"></button>
				<div id="sezzle-modal-core-content" class="sezzle-modal-content"></div>
			</div>
		</div>
	</div>
</div>
```

**Review by aholdahl — CHANGES_REQUESTED** [review]

**Review by aholdahl — APPROVED** [review]

**Pull request merged**